src/features/todos.ts

```diff
@@ -1,9 +1,24 @@
 import { Todo } from '../types/Todo';
 
-export const actions = {};
+type LoadTodos = { type: 'todos/LOAD', payload: Todo[] };
 
-const todosReducer = (): Todo[] => {
-  return [];
+type Action = LoadTodos;
+
+const loadTodos = (todos: Todo[]): Action => ({
+  type: 'todos/LOAD',
+  payload: todos,
+});
+
+export const actions = { loadTodos };
+
+const todosReducer = (todos: Todo[] = [] || null, action: Action) => {
```

Comment:
The initial state of todos is set to [] || null, which is essentially []. Null will never be used because [] is a truthy value in JavaScript. If you want to have null as a possible initial state, consider revising this.